rails/client/package.json

```diff
@@ -19,7 +19,7 @@
     "antd": "^2.7.4",
     "classnames": "^2.2.5",
     "humps": "^2.0.1",
-    "kaiju-bundle": "git+https://github.com/StephenEsser/kaiju-bundle.git",
+    "kaiju-bundle": "git+https://github.com/StephenEsser/kaiju-bundle.git#kaiju-Forms",
```

Comment:
I'll revert this back to master after these changes have been approved. 

Because kaiju-bundle is not currently versioned I didn't want to check these changes in until both the application and bundle were in sync.